panther_driver/src/driver_node.py

```diff
@@ -280,7 +295,7 @@ def _decode_fault_flag(self, flag_val: int) -> FaultFlag:
         ref_flag_val = 0b00000001
 
         msg = FaultFlag()
-        msg.can_net_err = self._panther_can.can_connection_correct()
+        msg.can_net_err = not self._panther_can.can_connection_correct()
```

Comment:
`can_connection_correct()` method is used only for detecting errors and each time is preceded by `not`. Maybe it would be better to rename it `can_connection_error()` and update returned boolean stating?